guides/template.adoc

```diff
@@ -0,0 +1,41 @@
+= TITLE OF THE GUIDE
+:summary: ONELINE EXPLANATION OF THE GUIDE
+:includedir: _includes
+include::{includedir}/_attributes.adoc[]
+// you can override any attributes eg to lengthen the 
+// time to complete the guide
+:prerequisites-time: 10
+
+TODO: Add a summary of what you achieve in this guide.
+Something like:
+
+In this guide, you will learn how to setup and use Eclipse MicroProfile Config in your application.
+
+include::{includedir}/_prerequisites.adoc[]
+* Any additional prerequisites specific to this guide (eg you have completed a previous guide)
```

Comment:
We may need a new line before this for the * to be rendered as a bullet.

Reply:
Also, maybe a TOC would be nice to have. 
I think `toc::[]` can be used to add it automatically.